omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ModuleActionController1_8Tests.java

```diff
@@ -94,6 +97,21 @@ public void shouldDoNothingIfAtlasModuleAlreadyStarted() throws Exception {
 	}
 	
 	@Test
+	public void shouldUpdateModule() throws Exception {
+		String currentVersion = atlasModule.getVersion();
+		// To confirm the current version
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject module = deserialize(handle(req));
+		Assert.assertEquals(currentVersion, module.get("version"));
+
+		SimpleObject updatedModule = deserialize(handle(newPostRequest(getURI(), "{\"action\":\"update\", \"modules\":[\""
+		        + getUuid() + "\"], " +
+		        "\"downloadUrls\":[\"https://bintray.com/openmrs/omod/download_file?file_path=atlas-2.2.omod\"]}")));
```

Comment:
Does this mean the test requires an internet connection to pass? I think you need to mock the end point to return this file, so the file would need to be a resource that is part of the module.